.github/workflows/release.yml

```diff
@@ -1,24 +1,49 @@
 name: Release
 
 on:
+  workflow_dispatch:
   push:
     tags:
       - "v*"
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
```

Comment:
Cross compilation is possible on a single machine. We can only use linux.

Reply:
Right - I tried this locally as well although it works, cross compilation takes a while. It's not a big deal.